pkg/identityserver/user_registry_test.go

```diff
@@ -54,8 +54,8 @@ func TestValidatePasswordStrength(t *testing.T) {
 			"password1":    false, // Too common.
 			"āaaAAA123!@#": true,
 			"       1A":    true,
-			"āaa	AAA123 ":  true,
-			"āaaAAA123 ":   true,
+			"āaa	AAA123 ": true,
```

Comment:
Don't try to fight this - `gofumpt` had a release yesterday and the installation procedure is not stable due to `GOPROXY` - sometimes you get the latest version in CI, sometimes you get the old version. 

On your local machine just do `./tools/bin/mage go:fmt`.

Reply:
OK I dropped the commit. I'll merge even with the Go Code failure.